system/jlib/jlog.hpp

```diff
@@ -1107,6 +1107,8 @@ extern jlib_decl void UseSysLogForOperatorMessages(bool use=true);
 #define SYSLOG querySysLogEventLogger()->log
 #define AUDIT SYSLOG                               // bwd compatibility
 
+#define SLAVEIDSTR "SLAVEID" //Not sure this should be here
```

Comment:
really need to move this somewhere common to workunit and thor code, but this is too specific to be jlib.
Also true of "thormaster" and similar strings used between components.
There may be a convenient thor include to use somewhere, perhaps thor.hpp.

Reply:
How about move it to thorhelper.hpp?

Reply:
yep, that's fine.